packages/tasks/src/pipelines.ts

```diff
@@ -585,7 +585,7 @@ export const PIPELINE_DATA = {
 	},
 	"text-to-video": {
 		name: "Text-to-Video",
-		modality: "cv",
+		modality: "multimodal",
```

Comment:
This was done in https://github.com/huggingface/huggingface.js/pull/477 as part of existing multimodal discussions in Slack (search for this PR). I'm happy to reconsider this specific case

Reply:
For me, multimodal is anything that is not single modality

* If only images are involved => cv
* if only text is involed => nlp
* if image + text, or any other combination of modalities (like text and video) => multimodal

Reply:
Image generation and captioning involve both text and images. I think the discussion had converged in cardinality of modalities of input or output. Please refer to the existing discussions